app/models/user.rb

```diff
@@ -11,6 +11,7 @@ class User < ActiveRecord::Base
 
   has_many :approvals
   has_many :observations
+  has_many :observers, through: :observers, source: :user
   has_many :comments
```

Comment:
oops! meant for this to be `through: :observations`...wonder how that was even working!?